tests/integration/targets/azure_ops_test_azure_manage_snapshot/playbooks/main.yml

```diff
@@ -0,0 +1,59 @@
+---
+- name: Setup virtual host
+  hosts: localhost
+  connection: local
+  gather_facts: false
+  tasks:
+    - name: Include default variables
+      ansible.builtin.include_vars:
+        file: ../vars/vars.yml
```

Comment:
You could use defaults/main.yml for default variables, so you don’t need an additional task.

Reply:
I cannot use defaults/main.yml because this is running as a playbook.